secv_guis/base_widgets.py

```diff
@@ -352,27 +377,24 @@ def slider_to_p_val(self, sl_val):
         """
         delta = float(sl_val) / self.thresh_num_steps
         pval = self.thresh_min + delta * (self.thresh_max - self.thresh_min)
-        #print("pval ",pval)
         return pval
 
-    def _set_thresh_label(self, lbl, sl_val):
+    def _set_thresh_label(self, lbl, sl_val,name):
```

Comment:
linting: space after comma! check flake8, you can configure your text editor to automatically let you know about linting "errors", after some time you end up writing code in standard style